### DISCUSSION TITLE
Modal/Flyout overlay

### DISCUSSION THREAD

**Discussion by codeldev** [livewire/flux#163] [Feature Requests] [answered]
Does anyone know if it's possible to change the style of the overlay on modals & flyouts?

I'd like to darken the overlay a little more and add some slight background blur. (And yes I looked at the flux components already, but could not see anything related to the overlay)

If anyone has suggestions, I'd be grateful. Thanks!

**Comment by danyderigon** [comment, marked as answer]
Hello.

Try this in your `resources/css/app.css`

```
[data-flux-modal] > dialog::backdrop {
    background-color: rgba(0, 0, 0, 0.4) !important;
    backdrop-filter: blur(1px);
}
```

https://fluxui.dev/docs/customization#global-style-overrides

**Comment by codeldev** [comment]
Thanks @danyderigon 

Implementing that worked great!

```
@layer components
{
    [data-flux-modal] > dialog::backdrop
    {
        @apply bg-black/60 backdrop-blur;
    }
}
```
